tests/kernel/workq/work/src/main.c

```diff
@@ -244,6 +252,21 @@ static void test_queue_start(void)
 		zassert_equal(strcmp(tn, cfg.name), 0, NULL);
 	}
 
+	cfg.name = NULL;
+	zassert_equal(invalid_test_queue.flags, 0, NULL);
+	k_work_queue_start(&invalid_test_queue, invalid_test_stack, STACK_SIZE,
+			    PREEMPT_PRIORITY, &cfg);
+	zassert_equal(invalid_test_queue.flags, K_WORK_QUEUE_STARTED, NULL);
+
+	if (IS_ENABLED(CONFIG_THREAD_NAME)) {
+		const char *tn = k_thread_name_get(&invalid_test_queue.thread);
+
+		printk("%s\n", tn);
+		zassert_true(tn != cfg.name, NULL);
```

Comment:
You don't need this `printk`. If you need to print something please use `TC_PRINT`.

Reply:
Sorry for that, I will remove that and update in latest change, thank for pointing it out.